app/styles/main.scss

```diff
@@ -8,3 +8,4 @@
 
 @import "layout";
 @import "headers/header";
+@import "headers/fifth";
```

Comment:
please, import `_fifth.scss` file into `headers/header`. 
`headers/header` is file with all the headers imports. So in `main.scss` we can only import one `headers/header` with all the headers styles. 
And please always add empty line at the end of the file. Otherwise github shows you this red small arrow

Reply:
I corrected this.